app/seeders/user_seeder.rb

```diff
@@ -0,0 +1,26 @@
+class UserSeeder
+  USERS = [
+    {
+      username: "member",
+      email: "member@member.com",
+      password: "password",
+      first_name: "Member",
+      last_name: "User",
+      role: "member"
+    },
+     {
+      username: "admin",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.